src/bootstrap-table.js

```diff
@@ -192,6 +192,10 @@ class BootstrapTable {
       })
     })
 
+    if (this.options.data.length === 0) {
```

Comment:
Not sure why this line fix the sync between two tables

Reply:
I'm also not sure, but without that javascript will use a shared array.
You also dont have the issue if one of the table have data (one row is enough), because there use not shared arrays.

Reply:
So, if we change something in this.options both tables will be affected?

Reply:
This only occurs in this edge case (Both table has no init data(rows) also this only happens with `this.options.data`.

Reply:
ok, thanks @UtechtDustin . @wenzhixin do you have any idea? I think the fix is good because it is working now but it looks to me like a workaround

Reply:
Just a explanation for you @djhvscf, if you do
```
let x = []
let y = x
y.push('test')
alert(x)
```
Example: https://live.bootstrap-table.com/code/UtechtDustin/4659

You will see `test`, because javascript dont copy the array. Instead of a copying, javascript will set the internal reference/pointer of the array (variable x) to variable y. Which means both variables share the same array.

I'm not sure where this happens in the bootstrap-table so this PR is just a workaround to prevent bugs.

Reply:
Yes, i know that. But we should remove those kind of problems since every table should be isolated

Reply:
oh okay, i thought you dont know that because of you question :D
> So, if we change something in this.options both tables will be affected?

Reply:
Thanks, @UtechtDustin and @djhvscf , I will check this problem.